test/axes/timeAxisTests.ts

```diff
@@ -294,6 +294,38 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels show correctly when display format is set to 'center'", () => {
+    let svg = TestMethods.generateSVG(400, 100);
+    scale.domain([new Date("2015-09-02"), new Date("2015-09-03")]);
```

Comment:
Sorry for jumping in so late. This dates are not formatted properly. That might be why Firefox was behaving differently before.

Chrome
<img width="296" alt="screen shot 2015-09-15 at 2 41 28 pm" src="https://cloud.githubusercontent.com/assets/3248682/9886460/667d9db0-5bb8-11e5-9b50-5e83465a2a15.png">

Firefox
<img width="228" alt="screen shot 2015-09-15 at 2 41 34 pm" src="https://cloud.githubusercontent.com/assets/3248682/9886473/73d3e9a6-5bb8-11e5-9563-17e9986fb103.png">

Proper way of doing it
```typescript
new Date(2015, 08, 02)
```